### PR TITLE
added information related to the metrochart.js library

### PR DESCRIPTION
**Pull request by jspaaks**
@blootsvoets I assigned you for the PR, but feel free to reassign as appropriate.

### CODE REVIEW

**Comment by jspaaks** [comment]
Uggh...errors of course. Hold your horses I'm on it

File: _person/j.spaaks.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ownerOf:
+
+contributorOf:
+- http://software.esciencecenter.nl/software/metrochart
+userOf:
+
+engineerOf:
+
+---
+Jurriaan description TODO
```
Comment (blootsvoets) [inline _person/j.spaaks.md]: Please add.

File: _project/sherlock.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: "sherlock"
+uses:
+- http://software.esciencecenter.nl/software/metrochart
+involvedOrganization:
+- http://software.esciencecenter.nl/organization/nfi
+---
+
+
```
Comment (blootsvoets) [inline _project/sherlock.md]: please add

File: _software/metrochart.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+programmingLanguage:
+- TypeScript
+license:
+- apache-2.0
+competence:
+- Big Data Analytics
+discipline:
+- eScience Methodology
+expertise:
+- Orchestrated Computing
```
Comment (blootsvoets) [inline _software/metrochart.md]: Rather visualisation, right? Orchestrated computing is a branch of Distributed and Efficient Computing

File: _person/j.spaaks.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+ownerOf:
+
+contributorOf:
+- http://software.esciencecenter.nl/software/metrochart
+userOf:
+- http://software.esciencecenter.nl/software/metrochart
+engineerOf:
+- http://software.esciencecenter.nl/project/sherlock
+---
+Jurriaan description TODO
```
Comment (blootsvoets) [inline _person/j.spaaks.md]: TODO?

**Pull request merged**